apps/dashboard/src/app/components/messages/message-modal.component.scss

```diff
@@ -0,0 +1,2 @@
+@import 'variables';
+@import '../../../../../../libs/pastanaga-angular/projects/pastanaga-angular/src/styles/theme/_modal.tokens.scss';
```

Comment:
why do you need this import? `_modal.tokens ` is not already imported in `variables`?